redaxo/src/addons/be_style/install.php

```diff
@@ -16,5 +16,8 @@
 $files = require __DIR__.'/vendor_files.php';
 
 foreach ($files as $source => $destination) {
-    rex_file::copy($addon->getPath($source), $addon->getAssetsPath($destination));
+    if (rex_file::copy($addon->getPath($source), $addon->getAssetsPath($destination)) === false) {
```

Comment:
```suggestion
    if (false === rex_file::copy($addon->getPath($source), $addon->getAssetsPath($destination))) {
```

Ich bevorzuge mittlerweile yoda conditions überall. Finde, dass man so viel schneller erkennt, was für eine Bedingung es ist.
Also so sehe ich nun direkt, dass `rex_file::copy` mit `false` gecheckt wird.
Bei der Ursprungsvariante sehe ich hingegen nicht so schnell, ob es ein Positiv- oder Negativ-Check ist.